core/src/main/java/hudson/util/FormFieldValidator.java

```diff
@@ -622,9 +622,9 @@ protected void check() throws IOException, ServletException {
                     return;
                 }
                 
-                com.trilead.ssh2.crypto.Base64.decode(v.toCharArray());
+                java.util.Base64.getDecoder().decode(v);
```

Comment:
I would rather hide this method in an util method, which returns a checked exception